static/app/utils/replays/mergeSpanEntries.tsx

```diff
@@ -0,0 +1,15 @@
+import type {RawSpanType} from 'sentry/components/events/interfaces/spans/types';
+import {Entry, EntryType, Event} from 'sentry/types/event';
+
+/**
+ * Merge all spans from each Event in the `events` array
+ */
+export default function mergeSpanEntries(events: Event[]): Entry {
+  const spans = events.flatMap(event =>
+    event.entries.flatMap((entry: Entry) =>
+      entry.type === EntryType.SPANS ? (entry.data as RawSpanType[]) : []
+    )
+  );
+
+  return {type: EntryType.SPANS, data: spans};
+}
```

Comment:
This is also copy+pasted from [static/app/views/replays/utils/mergeEventsWithSpans.tsx](https://github.com/getsentry/sentry/blob/master/static/app/views/replays/utils/mergeEventsWithSpans.tsx) but returns an Entry instead of an Event. 

We return an Entry now so that we can compose multiple Entries into a single Event inside of the Replay class.